src/plugins/walletConnectToDapps/v2/components/modals/EIP155TransactionConfirmation.tsx

```diff
@@ -46,10 +47,11 @@ export const EIP155TransactionConfirmation: FC<
 
   const form = useForm<CustomTransactionData>({
     defaultValues: {
-      nonce: transaction.nonce ? convertHexToNumber(transaction.nonce).toString() : undefined,
-      gasLimit: transaction.gasLimit
-        ? convertHexToNumber(transaction.gasLimit).toString()
-        : undefined,
+      nonce: transaction?.nonce ? convertHexToNumber(transaction.nonce).toString() : undefined,
+      gasLimit:
+        transaction?.gasLimit ?? transaction?.gas
```

Comment:
Some messages may contain `gas` instead of `gasLimit` as tested with Uniswap. Confirmed this is the same by simulating in tenderly as well as broadcasting the signed message with chain-adapters